myrmidon.sh

```diff
@@ -1,28 +1,52 @@
-#!/bin/bash
-cwd=$(echo $(dirname $0))
-
-# Use ~/.myrmidon-tasks.json as default, otherwise use incoming path
-config_file="${1:-"$HOME/.myrmidon-tasks.json"}"
-tasks=$(cat $config_file)
-
-# Pass tasks to rofi, and get the output as the selected option
-selected=$(echo $tasks | jq -j 'map(.name) | join("\n")' | rofi -dmenu -matching fuzzy -i -p "Search tasks")
-task=$(echo $tasks | jq ".[] | select(.name == \"$selected\")")
-
-# Exit if no task was found
-if [[ $task == "" ]]; then
-  echo "No task defined as '$selected' within config file."
-  exit 1
-fi
-
-task_command=$(echo $task | jq ".command")
-confirm=$(echo $task | jq ".confirm")
-
-# Check whether we need confirmation to run this task
-if [[ $confirm == "true" ]]; then
-  # Chain the confirm command before executing the selected command
-  confirm_script="$cwd/confirm.sh 'Confirm $selected?'"
-  eval "$confirm_script && \"$task_command\" > /dev/null &"
-else
-  eval "\"$task_command\" > /dev/null &"
-fi
\ No newline at end of file
+#!/usr/bin/env bash
+
+function func_rofi_confirm() {
+  local OPTIONS UNSAFE RESPONSE
+
+  [[ ${MESSAGE} == "null" ]] && MESSAGE="Confirm '${SELECTED}' ?"
+  OPTIONS="No\\nYes";
+
+  UNSAFE=$(echo "${TASK}" | jq ".unsafe")
+  [[ ${UNSAFE} == "true" ]] && OPTIONS="Yes\\nNo";
```

Comment:
Is this to provide inverted options, right? default to Yes instead of No?
If so, the naming seems a bit backwards to me. like this should be the safe, unsafe should be the original behavior.

Maybe this could be set as a different name associated with confirm, maybe
```
confirm: true
confirmInvert: true
```

I'm not sure I like it though!

Reply:
Another option could be setting confirm to be either true or an object with options, for instance:

```
confirm: {
   message: "This is dangerous! are you sure",
   invert: true
}
```
This makes it more explicit what those properties do, which are directly related to the `confirm` mechanism.

Reply:
the idea behind the `unsafe` keyword actually comes from... my camera's settings :)
by default when deleting a picture, there's a yes/no confirmation with the focus on the `no`
there's a setting in the menu that they called 'unsafe deleting' that makes the 'yes' selected by default instead.
so yeah. i guess `confirmInvert` makes more sense to most people.
a whole confirm object is also an interesting idea !